assets/xml/objects/object_sdn.xml

```diff
@@ -1,66 +1,66 @@
-        <Limb Name="object_sdn_Standardlimb_00D5B8" Type="Standard" Offset="0xD5B8" />
-        <Limb Name="object_sdn_Standardlimb_00D5C4" Type="Standard" Offset="0xD5C4" />
-        <Limb Name="object_sdn_Standardlimb_00D5D0" Type="Standard" Offset="0xD5D0" />
-        <Limb Name="object_sdn_Standardlimb_00D5DC" Type="Standard" Offset="0xD5DC" />
-        <Limb Name="object_sdn_Standardlimb_00D5E8" Type="Standard" Offset="0xD5E8" />
-        <Limb Name="object_sdn_Standardlimb_00D5F4" Type="Standard" Offset="0xD5F4" />
-        <Skeleton Name="object_sdn_Skel_00D640" Type="Flex" LimbType="Standard" Offset="0xD640" />
-        <Animation Name="object_sdn_Anim_00DC7C" Offset="0xDC7C" />
+        <DList Name="gDemoheishiBottleDL" Offset="0x630" />
+        <DList Name="gDemoheishiBottleBaseDL" Offset="0x768" />
+        <DList Name="gDemoheishiEmptyBottleSpaceDL" Offset="0x800" />
+        <Texture Name="gDemoheishiBottleTex" OutName="tex_000808" Format="i4" Width="16" Height="16" Offset="0x808" />
+        <Animation Name="gDemoheishiStandWithHandOnChest" Offset="0xA54" />
+        <Animation Name="gDemoheishiDrink" Offset="0x20D8" />
+        <Animation Name="gDemoheishiCheerWithSpear" Offset="0x2A84" />
+        <Animation Name="gDemoheishiSitAndReach" Offset="0x3380" />
+        <Animation Name="gDemoheishiWave" Offset="0x3BFC" />
+        <Animation Name="gDemoheishiStandUp" Offset="0x4770" />
+        <Animation Name="gDemoheishiLookDown" Offset="0x4AC0" />
+        <Animation Name="gDemoheishiComeUpHere" Offset="0x5320" />
+        <Animation Name="gDemoheishiHalt" Offset="0x57BC" />
+        <Animation Name="gDemoheishiHalt2" Offset="0x5D28" />
+        <Animation Name="gDemoheishiHalt3" Offset="0x64C0" />
+        <Animation Name="gDemoheishiStandHandOnHip" Offset="0x6C18" />
+        <DList Name="gDemoheishiTorsoDL" Offset="0x9EB0" />
+        <DList Name="gDemoheishiHeadDL" Offset="0xA1A8" />
+        <DList Name="gDemoheishiRightShoulderDL" Offset="0xA678" />
+        <DList Name="gDemoheishiRightForearmDL" Offset="0xA768" />
+        <DList Name="gDemoheishiRightHandWithSpearDL" Offset="0xA9A8" />
+        <DList Name="gDemoheishiLeftShoulderDL" Offset="0xADE0" />
+        <DList Name="gDemoheishiLeftForearmDL" Offset="0xAED0" />
+        <DList Name="gDemoheishiLeftHandDL" Offset="0xB120" />
+        <DList Name="gDemoheishiPelvisDL" Offset="0xB438" />
+        <DList Name="gDemoheishiRightThighDL" Offset="0xB5B0" />
+        <DList Name="gDemoheishiRightCalfDL" Offset="0xB698" />
+        <DList Name="gDemoheishiRightFootDL" Offset="0xB988" />
+        <DList Name="gDemoheishiLeftThighDL" Offset="0xBAD0" />
+        <DList Name="gDemoheishiLeftCalfDL" Offset="0xBBB8" />
+        <DList Name="gDemoheishiLeftFootDL" Offset="0xBEB8" />
+        <Texture Name="gDemoheishiSkinTlut" OutName="tlut_00C000" Format="rgba16" Width="16" Height="16" Offset="0xC000" />
```

Comment:
```suggestion
        <Texture Name="gDemoheishiSkinTLUT" OutName="demoheishi_skin_tlut" Format="rgba16" Width="16" Height="16" Offset="0xC000" />
```
TLUT (Texture Look Up Table) is an acronym so usually do all caps for it. Also don't forget to update the `OutName` as well for the textures.

Reply:
Fixed, thanks for catching this